app/models/crop.rb

```diff
@@ -33,6 +33,47 @@ class Crop < ActiveRecord::Base
       :with => /^https?:\/\/en\.wikipedia\.org\/wiki/,
       :message => 'is not a valid English Wikipedia URL'
     }
+  
+  include Elasticsearch::Model
+  include Elasticsearch::Model::Callbacks
+  index_name [Rails.env, "growstuff"].join('_')
+  settings index: { number_of_shards: 1},
+    analysis: {
+      tokenizer: {
+        gs_edgeNGram_tokenizer: {
+          type: "edgeNGram",
+          min_gram: 2,
+          max_gram: 20,
+          token_chars: [ "letter", "digit" ]
+        }
+      },
+      analyzer: {
+        gs_edgeNGram_analyzer: {
+          tokenizer: "gs_edgeNGram_tokenizer",
+          filter: ["lowercase"]
+        }
+      },
+    } do
+    mappings dynamic: 'false' do
+      indexes :id, type: 'long'
```

Comment:
I would include `created_at`, `updated_at`, `slug`, and `creator_id` in the index as well, since we will almost certainly want to search against those in fancier searches soon.